configure.py

```diff
@@ -1189,6 +1189,7 @@ def find_ninja():
                 'rust/wasmtime_bindings/src/lib.rs',
                 'utils/to_string.cc',
                 'service/topology_state_machine.cc',
+                'service/topology_mutation.cc',
```

Comment:
question: There is also a CMakeLists.txt file in the `service` directory. If it's still used, I think it might be good to list the file there too